source/prerequisites/my-sql.txt

```diff
@@ -10,4 +10,92 @@ MySQL Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-MySQL.
\ No newline at end of file
+To run sync jobs from a MySQL source database, the database may require 
+some configuration changes. If Relational Migrator determines the 
+database needs configuration changes, it automatically generates a 
+SQL script with the required changes. It is recommended to have a 
+Database Administrator (DBA) review the commands in this script and 
+perform their execution on the database server. This topic 
+provides more details on the required configuration steps.
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+For details on supported versions of MySQL, see 
+:ref:`supported-databases`.
+
+About this Task
+---------------
+
+Connecting Relational Migrator to MySQL 
+requires the `binary log <https://dev.mysql.com/doc/refman/8.0/en/binary-log.html>`__ 
+enabed on your MySQL instance. The binary log (Binlog) records all 
+operations in the order they are committed to the database. 
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: (Optional) User permissions setup 
+
+      Relational Migrator generates SQL that can be executed to enable 
+      ``Binlog`` for the source MySQL database. The following 
+      code creates a new MySQL service account for Relational Migrator 
+      to connect to the MySQL instance. 
+
+      Create a service account:
+
+      .. code-block:: sql
+         :copyable: true
+
+         CREATE USER 'user'@'localhost' IDENTIFIED BY 'password';
+
+      Grant the required permissions to the service account:
+
+      .. code-block:: sql
+         :copyable: true
+         
+         GRANT SELECT, RELOAD, SHOW DATABASES, REPLICATION SLAVE, REPLICATION CLIENT ON *.* TO 'user'@'%';
+
+      Apply the user privilege changes:
+
+      .. code-block:: sql
+         :copyable: true
+
+         FLUSH PRIVILEGES; 
+
+   .. step:: Manually verify Binlog is enabled
```

Comment:
Need to describe how to turn it on if it is off.

Reply:
Also we should add a note that binlog is on by default in MySQL 8.x, but not in 5.x.

Reply:
Added a additional section[ Locate and Update the MySQL Configuration File](https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-34352/prerequisites/my-sql/#locate-and-update-the-mysql-configuration-file) and I added a note about the binlog being on by default and additionally a AWS RDS MySQL note about enabling automated backups per [step 4 of the Debezium docs](https://debezium.io/documentation/reference/stable/connectors/mysql.html#enable-mysql-binlog).